samples/nextjs/package.json

```diff
@@ -10,7 +10,7 @@
     ],
     "sitecoreDistPath": "/dist/JssNextWeb",
     "sitecoreConfigPath": "/App_Config/Include/zzz",
-    "graphqlEndpointPath": "/sitecore/api/graph/edge",
+    "graphQLEndpointPath": "/sitecore/api/graph/edge",
```

Comment:
Right now, we figure out the graphQL endpoint by concatenating the Sitecore API hostname and this relative graphQLEndpointPath. Would there ever be a case when someone would need to use an absolute path for 'graphQLEndpointPath'?  For example, what about using both, SSR and SSG for a hybrid app - is it still a single endpoint that's smart enough to forward certain requests to the CD, or would we need to support using 2 endpoints?

Reply:
@anastasiya29 We may have to get more sophisticated with the `graphQLEndpoint` generation once we implement Exp Edge integration for sure. However, I'm not sure why the endpoint would differ between SSR / SSG. Either way, seems out of scope of this task.